en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md

```diff
@@ -14,16 +14,57 @@ Let's learn how Sam implements this!
 
 Follow the steps below to configure account locking due to failed login attempts.
 
-1.	Open the `deployment.toml` file in the `<IS_HOME>/repository/conf` directory and check whether the following listener configs are in place.
-
-	```
-	[event.default_listener.identity_mgt]
-	priority= "50"
-	enable = false
-	[event.default_listener.governance_identity_mgt]
-	priority= "95"
-	enable = true
-	```
+1.	Open the `deployment.toml` file in the `<IS_HOME>/repository/conf` directory.
+
+	1.	To request password entry from the users who are registered via Management Console, add the following configurations. 
+
+
+	 	```
+	  	[event.default_listener.identity_mgt]
+	  	priority= "50"
+	  	enable = false
+	  	event.default_listener.governance_identity_mgt]
+	    priority= "95"
```

Comment:
does this render properly?

Reply:
<img width="768" alt="Render-display" src="https://user-images.githubusercontent.com/40081059/167310722-49c425b9-de6c-4a6a-91ab-be8e52ae9281.png">


**The section is rendered properly**